### DISCUSSION TITLE
New signup/signin page

### DISCUSSION THREAD

**Discussion by dev13-suthar** [lukevella/rallly#1395] [Ideas]
Right now Signin/Signup pages are simple card it would be better an  idea to keep user discovering About app,  we can add  some more content about the rallly just like cal.com or dub.sh. Any figma available?

**Comment by dev13-suthar** [comment]
@lukevella is repo being maintained or not?. looking forward for contributing

**Reply by lukevella** [reply]
Yes it is. I need to manage my limited time and don't reply to everything straight away. Your idea is good but unless you are a designer and able to submit your own designs then it's not actionable so this is not a priority for me.
